File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver import ActionChains
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.common.by import By

service = webdriver.FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("The title is : ", driver.title)
    action = ActionChains(driver)
    action.click().perform()
    sideText = driver.find_element(By.CLASS_NAME, "active")
    print("After left click, the test is : ", sideText.text)
    action.double_click().perform()
    sideText = driver.find_element(By.CLASS_NAME, "active")
    print("After double click, the test is : ", sideText.text)
    action.context_click().perform()
    sideText = driver.find_element(By.CLASS_NAME, "active")
    print("After right click, the test is : ", sideText.text)
    driver.quit()

File: Selenium/Activities/Activity5.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

import javax.swing.*;

public class Activity5 {

    public static void main(String[] args){

        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");

        WebDriverManager.firefoxdriver().setup();

        WebDriver driver = new FirefoxDriver();

        driver.get("https://v1.training-support.net/selenium/input-events");

        System.out.println("The title of the page is : " + driver.getTitle());

        Actions action = new Actions(driver);
        action.click().pause(2000).build().perform();
        String sideText = driver.findElement(By.className("active")).getText();
        System.out.println("After left click, the test is : " + sideText);
        action .doubleClick().pause(2000).build().perform();
        sideText = driver.findElement(By.className("active")).getText();
        System.out.println("After double clicking, the text is : " + sideText);
        action.contextClick().pause(2000).build().perform();
        sideText = driver.findElement(By.className("active")).getText();
        System.out.println("After right click, the text is : " + sideText);

        driver.quit();

    }
}
